boa_cli/src/main.rs

```diff
@@ -2,29 +2,30 @@
 #![warn(clippy::perf)]
 #![allow(clippy::cognitive_complexity)]
 
+use boa::builtins::console::log;
 use boa::{exec::Executor, forward_val, realm::Realm};
+use std::io;
 use std::{fs::read_to_string, path::PathBuf};
 use structopt::StructOpt;
-
 /// CLI configuration for Boa.
 #[derive(Debug, StructOpt)]
 #[structopt(author, about)]
 struct Opt {
     /// The javascript file to be evaluated.
-    #[structopt(name = "FILE", parse(from_os_str), default_value = "tests/js/test.js")]
+    #[structopt(name = "FILE", parse(from_os_str))]
     files: Vec<PathBuf>,
-    /// Open a boa shell (WIP).
+    /// Open a boa shell.
```

Comment:
Does this option make sense now that shell is the default?

Reply:
It is now removed!